internal/cmd/init.go

```diff
@@ -45,6 +46,25 @@ func GetInitCmd() *cobra.Command {
 				return err
 			}
 
+			configFileExists, _ := filesystem.Exists(".openfeature.yaml")
+			if !configFileExists {
+				pterm.Info.Println("Creating .openfeature.yaml configuration file")
+				template := getConfigTemplate(flagSourceUrl)
+				err = filesystem.WriteFile(".openfeature.yaml", []byte(template))
+				if err != nil {
+					return err
+				}
+			} else if flagSourceUrl != "" {
+				pterm.Info.Println("Updating flag source URL in .openfeature.yaml", pterm.LightWhite(flagSourceUrl))
+				template := getConfigTemplate(flagSourceUrl)
+				err = filesystem.WriteFile(".openfeature.yaml", []byte(template))
```

Comment:
We may want to ask for confirmation before overriding an existing configuration. The user is running `init`, but I think we should follow a similar pattern to what we have above with the manifest itself.

Reply:
yea that's a great call, that way the human is in the loop and doesn't get surprised.